test/chopperbot/split_test.exs

```diff
@@ -42,5 +42,27 @@ defmodule Chopperbot.SplitTest do
                {"turbo", 353.1}
              ] == Split.apply_options(@sum_orders, ["+vat", "+service"])
     end
+
+    test "can apply a percentage discount to orders" do
+      assert [
+               {"kendo", 270.0},
+               {"neo", 360.0},
+               {"turbo", 270.0}
+             ] == Split.apply_options(@sum_orders, ["-10%"])
+    end
+
+    test "can apply a percentage addition to orders" do
+      assert [
+               {"kendo", 330.0},
+               {"neo", 440.0},
+               {"turbo", 330.0}
+             ] == Split.apply_options(@sum_orders, ["+10%"])
+    end
+
+    test "raises an error exception if an invalid option is given" do
+      assert_raise MatchError, fn ->
```

Comment:
FYI, an error of the previous version was `FunctionClauseError` 😅

Reply:
I see, it's okay. We can let it crashes for now 😂